### DISCUSSION TITLE
Extract all pages

### DISCUSSION THREAD

**Discussion by hypertyper84** [yakovmeister/pdf2image#187] [General]
Hi,

I am trying to extract all pages by setting the pageNumber property to -1 when using the fromBase64 method. Unfortunatelly, this ends up with the following error:
Error: Page number should be more than or equal 1

How to convert all pages from a base64 pdf string to an base64 png string?

Regards,
Robert

**Comment by mskec** [comment]
Please refer to the README for docs. If you believe there's a bug, please file a new issue with your code snippet.

https://github.com/yakovmeister/pdf2image#frombase64b64string-options
